contracts/Paper.sol

```diff
@@ -1,27 +1,34 @@
 pragma solidity ^0.8.4;
 
-contract Paper {
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Paper is Ownable {
     enum State {onReview, Approved, Rejected}
 
     struct FeedBack {
         string feedback;
         address reviewer;
     }
 
+    address private _owner;
+    bool private primaryChecked;
     string public ipfsHash;
     string[] public fields;
     State public paperState;
-    address private owner;
+    address private author;
     address[] private reviewers;
     State[] private reviewStates;
     FeedBack[] private feedbacks;
+    FeedBack[] private pendingFeedbacks;
     uint private submitDate;
     uint private deadlineDate;
```

Comment:
I think all fields should be public, as the review process is supposed to be public.